oxide-auth-db/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+extern crate argon2;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+extern crate serde_json;
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate log;
+extern crate url;
+extern crate r2d2_redis;
+extern crate r2d2;
+extern crate reqwest;
+extern crate dotenv;
```

Comment:
With 2018 edition most of these are unecessary.

Reply:
That's true. Thanks. I've removed them.